packages/webapp/src/views/common/file-upload-check.html

```diff
@@ -39,13 +39,13 @@
             items: [
                 {
                     value: "yes",
-                    text: "Yes",
+                    text: "Yes, my local planning authority have approved this file",
                     id: "check-upload-correct-yes",
                     checked: yesSelection
                 },
                 {
                     value: "no",
-                    text: "No, choose a different file",
+                    text: "No",
```

Comment:
Same comment again: This is a common code file that is shared across all uploads, therefore this text change will affect all usage including landowner journey.  Please undo and ensure the text change only applies for the developer metric upload